### PR TITLE
Feat appointments

### PR DESCRIPTION
**Pull request by chrisT-pixel**
requesting a merge - feat_appointments to master

### CODE REVIEW

File: Start-Finance/InstaRichie/Package.appxmanifest [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
-  <Identity Name="32458featapps.InstaRichie" Publisher="CN=TAFE19S204" Version="3.1.20.0" />
+  <Identity Name="32458featapps.InstaRichie" Publisher="CN=Chris Taylor" Version="3.1.20.0" />
```
Comment (chrisT-pixel) [inline Start-Finance/InstaRichie/Package.appxmanifest]: Publisher name and TafeID

File: Start-Finance/InstaRichie/Views/TransactionPage.xaml [hunk 2]
```diff
@@ -102,6 +102,7 @@
             </controls:PageHeader.PrimaryCommands>
 
         </controls:PageHeader>
+
```
Comment (chrisT-pixel) [inline Start-Finance/InstaRichie/Views/TransactionPage.xaml]: I just created an extra blank line for readability

File: Start-Finance/InstaRichie/Start Finance.csproj [hunk 3]
```diff
@@ -22,7 +22,7 @@
     <AppxSymbolPackageEnabled>False</AppxSymbolPackageEnabled>
     <AppxBundle>Always</AppxBundle>
     <AppxBundlePlatforms>x86|x64|arm</AppxBundlePlatforms>
-    <PackageCertificateThumbprint>B73776EE1DED26D9509E97C09C92664BB245C3C8</PackageCertificateThumbprint>
```
Comment (chrisT-pixel) [inline Start-Finance/InstaRichie/Start Finance.csproj]: will this affect local compiler?

**Comment by FedorGabrus** [comment]
Should be ok.

Regards,
Fedor

**Comment by chrisT-pixel** [comment]
Cheers Fedor,

It's a nice little application - fun to work with.

Chris

**Pull request merged**